### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by Qiuhong-Wu**
Minor bug fix for Card Cultural Metropolis and Card Red Tourism Wave.

### CODE REVIEW

File: src/cards/turmoil/CulturalMetropolis.ts [hunk 1]
```diff
@@ -18,7 +18,8 @@ export class CulturalMetropolis implements IProjectCard {
 
     public canPlay(player: Player, game: Game): boolean {
         if (game.turmoil !== undefined) {
-            return game.turmoil.canPlay(player, PartyName.UNITY) && player.getProduction(Resources.ENERGY) >= 1;
+            //This card requires player has 2 delegates available
+            return game.turmoil.canPlay(player, PartyName.UNITY) && player.getProduction(Resources.ENERGY) >= 1 && (game.turmoil!.getDelegates(player.id) > 1 || (game.turmoil!.getDelegates(player.id) === 1 && game.turmoil?.lobby.has(player.id)))
```
[the remark this answers is not in the document]
Comment (jijiqw) [reply src/cards/turmoil/CulturalMetropolis.ts]: yeah, not need ! or ?

[the remark this answers is not in the document]
Comment (Qiuhong-Wu) [reply src/cards/turmoil/CulturalMetropolis.ts]: Thank you for your comments. I have changed it :)

File: src/cards/turmoil/CulturalMetropolis.ts [hunk 2]
```diff
@@ -27,12 +28,13 @@ export class CulturalMetropolis implements IProjectCard {
         player.setProduction(Resources.ENERGY,-1);
         player.setProduction(Resources.MEGACREDITS,3);
         game.addInterrupt(new SelectCity(player, game));
+        const title = "Select where to send two delegates";
         // To Do: Should allow use the free delegate (in lobby) for this card (only matter in very few secnarios)
```
Comment (nwai90) [inline src/cards/turmoil/CulturalMetropolis.ts:32]: @Qiuhong-Wu Let's remove this `To Do` comment as your PR fixes it

Comment (Qiuhong-Wu) [reply src/cards/turmoil/CulturalMetropolis.ts]: Done! Thanks :)

**Pull request merged**